### PR TITLE
Odin recipe

### PR DESCRIPTION
**Pull request by ayan-IN**
**Changes:**
1. Adding homepage with list of dishes or drinks to explore.
2. Each dishes or drink page is populated with : 
    (i) Image
    (ii) Description 
    (iii) Ingredients
    (iv) Steps to follow to make the dish or drink.
3. Demo link : [https://ayan-in.github.io/odin-recipes/](https://ayan-in.github.io/odin-recipes/)

### CODE REVIEW

File: recipes/Italian_Chicken_Marinade.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <ol>
+      <li>Whisk salad dressing, garlic powder, and salt together in a shallow baking dish; add chicken breasts and turn to coat. Cover the dish with plastic wrap and marinate in the refrigerator, 4 hours to overnight.</li>
+      <li>Preheat an outdoor grill for high heat and lightly oil the grate.</li>
+      <li>
+        Remove chicken from marinade and shake off excess; discard remaining marinade.
+      </li>
+      <li>Cook chicken on the preheated grill until no longer pink in the center and the juices run clear, about 7 to 8 minutes on each side. An instant-read thermometer inserted into the center should read at least 165 degrees F (74 degrees C).</li>
+    </ol>
+</body>
+</html>
```
Comment (roopaljasnani) [inline recipes/Italian_Chicken_Marinade.html:36]: All the files should have a new line at the end.

It's standard to have a blank LF/CRLF as a file terminator. When you will end up working on production-grade applications, linters associated with such applications will end up complaining about such seemingly trivial concerns. It is important therefore that you make it a habit to write clean code from the start.

One [helpful ref.](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3) that gives a gist of why new lines are a must

Pls make a habit of working hand-in-hand with helpful tools like [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) & [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) that come along with the VSCode editor & configure them to run automatically on save. It should save you from the hassle of manually making such or other formatting changes.

**Review by roopaljasnani — APPROVED** [review]
Code & PR LGTM!

One small nit: the Demo link is hyperlinked to an invalid url

**Comment by sankar-jtc** [comment]
Can you update the demo link? @ayan-IN

**Comment by ayan-IN** [comment]
Changes:
Fix of new line at the end of the file.
Update Demo Link.

**Review by roopaljasnani — APPROVED** [review]

**Pull request merged**